src/components/NavBar/SearchBarNft.css.ts

```diff
@@ -0,0 +1,242 @@
+import { style } from '@vanilla-extract/css'
+import { buttonTextSmall, subhead, subheadSmall } from 'nft/css/common.css'
+
+import { breakpoints, sprinkles, vars } from '../../nft/css/sprinkles.css'
+
+const DESKTOP_NAVBAR_WIDTH = 360
+const DESKTOP_NAVBAR_WIDTH_L = 440
+const DESKTOP_NAVBAR_WIDTH_XL = 540
+const DESKTOP_NAVBAR_WIDTH_XXL = 640
+
+const baseSearchStyle = style([
+  sprinkles({
+    paddingY: '12',
+    width: { sm: 'viewWidth' },
+    borderStyle: 'solid',
+    borderWidth: '2px',
+    borderColor: 'backgroundOutline',
+  }),
+  {
+    '@media': {
+      [`screen and (min-width: ${breakpoints.sm}px)`]: {
+        width: `${DESKTOP_NAVBAR_WIDTH}px`,
+      },
+      [`screen and (min-width: 1024px) and (max-width: 1080px)`]: {
+        width: `${330}px`,
+      },
+      [`screen and (min-width: 1190px) and (max-width: 1380px)`]: {
+        width: `${DESKTOP_NAVBAR_WIDTH_L}px`,
+      },
+      [`screen and (min-width: 1380px) and (max-width: 1479px)`]: {
+        width: `${DESKTOP_NAVBAR_WIDTH_XL}px`,
+      },
+      [`screen and (min-width: ${1480}px)`]: {
+        width: `${DESKTOP_NAVBAR_WIDTH_XXL}px`,
+      },
+    },
+  },
+])
+
+export const searchBarContainer = style([
+  sprinkles({
+    right: '0',
+    top: '0',
+    zIndex: '3',
+    display: 'inline-block',
+  }),
+  {
+    '@media': {
+      [`screen and (min-width: ${breakpoints.lg}px)`]: {
+        right: `-${DESKTOP_NAVBAR_WIDTH / 2}px`,
+        top: '-3px',
+      },
+      [`screen and (min-width: 1024px) and (max-width: 1080px)`]: {
+        right: `-${300 / 2}px`,
+      },
+      [`screen and (min-width: 1190px) and (max-width: 1380px)`]: {
+        right: `-${DESKTOP_NAVBAR_WIDTH_L / 2}px`,
+      },
+      [`screen and (min-width: 1380px) and (max-width: 1479px)`]: {
+        right: `-${DESKTOP_NAVBAR_WIDTH_XL / 2}px`,
+      },
+      [`screen and (min-width: ${1480}px)`]: {
+        right: `-${DESKTOP_NAVBAR_WIDTH_XXL / 2}px`,
+      },
+    },
+  },
+])
+
+export const searchBar = style([
```

Comment:
everything from here and below is the same.  I want to keep it simple, so I copied and pasted the file.  See above comment about removing flag, this should simplify